src/WPF/ArcGISRuntime.WPF.Viewer/Samples/MapView/DisplayGrid/DisplayGrid.xaml.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using System;
+using System.Windows;
+using Colors = System.Drawing.Color;
+
+namespace ArcGISRuntime.WPF.Samples.DisplayGrid
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Display a grid",
+        "MapView",
+        "Display and work with coordinate grid systems such as Latitude/Longitude, MGRS, UTM and USNG on a map view. This includes toggling labels visibility, changing the color of the grid lines, and changing the color of the grid labels.",
```

Comment:
I realize that this may have been handed down with the design, but the description is worded a little different than most. We usually start with something like "this sample illustrates/shows/etc ..."